templates/includes/sidebar.html

```diff
@@ -1,4 +1,20 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      <a href="{% url 'taxi:driver-detail' user.id %}">Driver: {{ user.first_name }}</a>
+    </li>
+    <li>
+      <a href="{% url 'logout' %}">Log out</a>
+    </li>
+    <br>
+    
+    {% else %}
+    <li>
+      <a href="{% url 'login' %}">Login</a>
```

Comment:
The 'login' URL is used here. Ensure that the URL pattern for login is correctly defined and that the corresponding view and template are set up.